structs.go

```diff
@@ -1566,15 +1566,15 @@ type UserGuildSettingsEdit struct {
 	ChannelOverrides     map[string]*UserGuildSettingsChannelOverride `json:"channel_overrides"`
 }
 
-// A GuildMemberEditData stores a payload to use in Session.GuildMemberEdit
+// A GuildMemberEditData stores a payload to use in Session.GuildMemberEditComplex
 // https://discord.com/developers/docs/resources/guild#modify-guild-member
 type GuildMemberEditData struct {
```

Comment:
To keep consistency between all edit structures I think it would be best to name it
```suggestion
type GuildMemberEdit struct {
```